vic/vic_run/src/lakes.eb.c

```diff
@@ -290,15 +290,16 @@ solve_lake(double           snowfall,
         *  6. Calculate initial energy balance over ice.
         **********************************************************************/
 
+        windi = (wind * log((2. + soil_con.snow_rough) / soil_con.snow_rough) /
+                 log(wind_h / soil_con.snow_rough));
+        if (windi < 1.0) {
+            windi = 1.0;
+        }
+
```

Comment:
Okay, I'm going to let you rework this then. Branch off this PR or open a PR of your own.